docs/searching.md

```diff
@@ -362,14 +362,14 @@ var addressFacetResults = results.GetFacet("Address"); // Returns the facets for
 
 Numeric range facets can be used with numbers and get facets for numeric ranges. For example, it would group documents of the same price range.
 
-There's two categories of numeric ranges - `DoubleRanges` and `Int64Range` for double/float values and int/long/datetime values respectively.
+There's two categories of numeric ranges - `DoubleRange` and `Int64Range` for double/float values and int/long/datetime values respectively.
```

Comment:
@nikcio under the hood `double/float` may be handled equivalently, but the examples show three different facet ranges `DoubleRange`, `FloatRange` and `Int64Range`.

Reply:
I have updated that 🙂